src/components/AdobeSignDialog.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <PvToast />
+  <PvDialog
+    v-model:visible="isVisible"
+    :pt="{ mask: { style: 'backdrop-filter: blur(2px);' } }"
+    :draggable="false"
+    :modal="true"
+    class="border-1h-3rem bg-white w-8"
+  >
+    <template #container>
+      <div v-if="!docCreated" class="bg-white p-3 border-1 border-round">
+        <p class="text-center">
+          To proceed, please enter your email address below. <br />
+          This will initiate an Adobe Sign workflow to securely obtain your signature on a required
+          {{ isAdult ? 'consent' : 'assent' }} form.
```

Comment:
```suggestion
          {{ props.isAdult ? 'consent' : 'assent' }} form.
```
I think we can use the value of the prop here.

Reply:
I believe that when we are on the html we don't need props.